packages/medusa/src/api/routes/admin/gift-cards/list-gift-cards.js

```diff
@@ -21,14 +22,23 @@ import { defaultFields, defaultRelations } from "./"
  */
 export default async (req, res) => {
   try {
+    const limit = parseInt(req.query.limit) || 50
+    const offset = parseInt(req.query.offset) || 0
+
```

Comment:
While these were not part of the initial feat, we should have some kind of pagenation going on